workshop/posting-error-example.md

```diff
@@ -0,0 +1,199 @@
+
+<img src = "screenshots/breakdown.png">
+
+### Where is this error occurring; what notebook/chunk?
+
+- In our workshop, notebook references tend to make sense to provide, however, more generally this should be any context around __where__ you've encountered this error.
+
+- If you are *not* working from any CCDL materials, it is *even more important* to provide context around what the notebook you are creating is supposed to be doing.
+
+- Note that line numbers change as you've added code, so line numbers as a reference aren't as helpful.
+
+*For our example:*  
+<img src = "screenshots/where-is-the-error.png" width = "600">
+
+### What is the goal of this code?  
+
+- Providing context around your end goals for this code will help others tailor their advice toward your goal as well as helping the understand what you are working toward.
+    *Examples of informatoni you may want to include (whenever it is applicable):*
+    - What is your scientific question?
+    - What does your input data look like?
+       - What kind of file *e.g.* `.TSV`, `.fastq`?
+       - What information is included the data?
+       - What are the columns?
+       - What are the rows?
+    - What'd your desired output?
+       - What kind of results are you looking for e.g. a plot, a table, processed/filtered data?
+       - What kind of file *e.g.* `.TSV`, `.fastq`?
+
+- Sometimes you can bypass an error completely by using an alternative strategy altogether, but your helper won't be able to suggest an alternative if they don't know your ultimate goal.
+
+*For our example:*  
+<img src = "screenshots/what-is-the-goal.png" width = "600">
+
+### What is the code that is producing this error?
+
+- You should [use backticks to format your code.](../virtual-setup/slack-procedures.md#adding-code-blocks-to-messages).
+Formatting your code makes it easier for others to distinguish between your code and the words in your post where you are otherwise describing the problem.
+
+- Screenshots are not ideal because others can't copy-paste your code from a screenshot. Being able to copy-paste the code with backticks makes it easiest for others to help you.
+
+*For our example:*  
+<img src = "screenshots/what-is-the-code.png" width = "600">
+
+### What is the error or problematic outcome?  
+
+- Include the any error messages verbatim. Also preferably not a screenshot so others can copy-paste for Google searching.
+- If you don't have an error message per se, but do have something else that is not working as expected, describe what isn't working and how you found it.
+     - *Example*: "This data.frame I'm showing in my code, called `cool_df` doesn't have row names when I look at it in my environment panel".
+- Context around when the error appears versus when it does not can be helpful clues to your reader.
+
+*For our example:*  
+<img src = "screenshots/what-is-the-code.png" width = "600">
```

Comment:
Looks like this is the same screenshot from the previous section, I would expect the error message in the screenshot here.

Reply:
Good catch!